src/Runner.java

```diff
@@ -1,38 +1,42 @@
 
+
 public class Runner {
 
 	public static void main(String[] args) {
 		Car[] cars = { new Car(1,  "Zaz",       "Slavuta",      2001, "Green", 5000,     1054576),
```

Comment:
If you use some values more then twice you should think about constant if you do it more - about enum. For example:
```CarType.ZAZ;```